extension/services/deezer/deezer.js

```diff
@@ -0,0 +1,80 @@
+import * as music from "../../intents/music/music.js";
+import * as serviceList from "../../background/serviceList.js";
+import * as content from "../../background/content.js";
+import { shouldDisplayWarning } from "../../limiter.js";
+
+class Deezer extends serviceList.Service {
+  async playQuery(query) {
+    await this.initTab("/services/deezer/player.js");
+    if(this.initTab){
+   window.alert ("Please signup or login to use this service. If you have, please ignore this message.");
+    }
+    await this.callTab("search", { query, thenPlay: true });
```

Comment:
So here there should be something like:

```
try {
  await this.callTab(...);
} catch (e) {
  if (e.message === "Not logged in") {
    e.displayMessage = "Please log in";
    this.context.activateTab(this.tab);
  }
  throw e;
}
```

And maybe this should go in `musicPlayer.js`